packages/wdio-utils/src/driver/utils.ts

```diff
@@ -84,10 +84,15 @@ export const downloadProgressCallback = (artifact: string, downloadedBytes: numb
         return
     }
     const percentage = ((downloadedBytes / totalBytes) * 100).toFixed(2)
-    log.info(`Downloading ${artifact} ${percentage}%`)
+    log.progress(`Downloading ${artifact} ${percentage}%`)
     lastTimeCalled = Date.now()
 }
 
+const _install = async (args: InstallOptions & { unpack?: true | undefined }) => {
+    await install(args)
+    log.progress('')
```

Comment:
Can you comment why we do this?

Reply:
Initially, `log.progress` is used within the `downloadProgressCallback` to show the download status. To ensure that subsequent log entries maintain their formatting, it's must to clear the `log.progress` output at the end.

Unfortunately, the `install` method doesn't include a built-in callback mechanism for this purpose. Consequently, we must manually invoke `log.progress` with an empty string to clear it after the `install` method is executed.

To avoid duplicating these lines of code, I've introduced this method.

Reply:
Can you comment this in the code just to document this?

Reply:
sure on it.